synapse/tests/test_daemon.py

```diff
@@ -56,6 +56,25 @@ async def test_dmon_ready(self):
                 async with await s_telepath.openurl(f'tcp://127.0.0.1:{port}/foo') as foo:
                     pass
 
+    async def test_dmon_ahainfo(self):
+
+        async with await s_daemon.Daemon.anit() as dmon:
+
+            host, port = await dmon.listen('tcp://127.0.0.1:0')
+            dmon.share('*', Foo())
+
+            async with await s_telepath.openurl(f'tcp://127.0.0.1:{port}') as proxy:
+                self.eq(proxy.ahainfo, {})
+
+        ahainfo = {'name': 'test.loop.vertex.link'}
+        async with await s_daemon.Daemon.anit(ahainfo=ahainfo) as dmon:
+
+            host, port = await dmon.listen('tcp://127.0.0.1:0')
+            dmon.share('*', Foo())
+
+            async with await s_telepath.openurl(f'tcp://127.0.0.1:{port}') as proxy:
+                self.eq(proxy.ahainfo, ahainfo)
```

Comment:
Can we add a `_getAhaInfo()` method to the Proxy and Client objects that will expose this information? That way we don't collide if someone implements a `ahainfo()` API on a endpoint.

Reply:
Or we could just store the data in an `self._ahainfo`  ( which we should do anyway to prevent the collision )